cinema_service/settings.py

```diff
@@ -20,9 +21,7 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = (os.getenv("DJANGO_SECRET_KEY"))
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
Security Warning: You shouldn't run with debug turned on in production! While it's okay to have `DEBUG = True` in your development settings, it's important to ensure this is turned off in your production settings. If `DEBUG` is `True`, then your app could potentially reveal sensitive information when it raises errors. You can use environment variables to manage your different configurations for development and production.